File: docs/reference/snippets/java/auth/account_login.java
OkHttpClient client = new OkHttpClient().newBuilder()
  .build();
MediaType mediaType = MediaType.parse("application/json");
RequestBody body = RequestBody.create(mediaType, "{\n    \"username\": \"cooluser\",\n    \"password\": \"YipohBe7quoohuev\"\n}");
Request request = new Request.Builder()
  .url("https://openapi.surreal-ai.com/v1/auth/account.login")
  .method("POST", body)
  .addHeader("Content-Type", "application/json")
  .build();
Response response = client.newCall(request).execute();

File: docs/reference/snippets/javascript/auth/account_login.js
var myHeaders = new Headers();
myHeaders.append("Content-Type", "application/json");

var raw = JSON.stringify({
  "username": "cooluser",
  "password": "YipohBe7quoohuev"
});

var requestOptions = {
  method: 'POST',
  headers: myHeaders,
  body: raw,
  redirect: 'follow'
};

fetch("https://openapi.surreal-ai.com/v1/auth/account.login", requestOptions)
  .then(response => response.text())
  .then(result => console.log(result))
  .catch(error => console.log('error', error));
